src/main/java/oxgame/app/SymbolValidator.java

```diff
@@ -1,14 +1,17 @@
-package xogame.app;
+package oxgame.app;
 
 import java.util.function.Supplier;
 
 public class SymbolValidator {
     public static Symbol validateSymbol(Supplier<String> input) throws IllegalArgumentException {
         String potentialSymbol = input.get().toUpperCase();
-        if (potentialSymbol.equals("X") || potentialSymbol.equals("O"))
```

Comment:
for such repeated "magic numbers" like "0" here you sould use enums or at least private static final fields